floss/language/go/extract.py

```diff
@@ -0,0 +1,410 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+
+import re
+import sys
+import struct
+import logging
+import pathlib
+import argparse
+from typing import List, Iterable, Optional
+from itertools import chain
+
+import pefile
+
+from floss.results import StaticString, StringEncoding
+
+logger = logging.getLogger(__name__)
+
+MIN_STR_LEN = 6
+
+
+def extract_strings_from_import_data(pe):
```

Comment:
let's add type hints for new functions for better documentation and so we can use mypy to find issues